server/protocols/protocols-imap4/src/main/java/org/apache/james/imapserver/netty/IMAPServer.java

```diff
@@ -132,6 +134,7 @@ public Optional<OidcSASLConfiguration> getOidcSASLConfiguration() {
     private final ImapDecoder decoder;
     private final ImapMetrics imapMetrics;
     private final GaugeRegistry gaugeRegistry;
+    private final Set<ConnectionCheck> connectionCheckSet;
```

Comment:
```suggestion
    private final Set<ConnectionCheck> connectionChecks;
```

Here and everywhere.